src/webrtc/groupCall.ts

```diff
@@ -1629,6 +1632,7 @@ export class GroupCall extends TypedEventEmitter<
         if (this.stats === undefined) {
             const userID = this.client.getUserId() || "unknown";
             this.stats = new GroupCallStats(this.groupCallId, userID, this.statsCollectIntervalTime);
+            this.stats.setParticipants(this.participants);
```

Comment:
Just a note:
I try to avoid dependencies between the VoIP objects like `GroupCall`, `CallFeed`, ...  and the stats package for example with `GroupCallStats`. 
It's not that easy because the values have to be calculated somewhere. In addition we also want statistics based on VoIp objects. But many dependencies means `GroupCall` constantly wanting  update `GroupCallStats` .

To avoid complicated dependencies I installed the cascading event listener. So GroupCall can expand the Webrtc statistics with VoIP statistics. Here how i did it with CallFeed stats. This is also not optimal since ``CallFeedStatsReporter.expandCallFeedReport`` needs to know about `CallFeed` but it is less coupled. 

https://github.com/matrix-org/matrix-js-sdk/blob/f0673101981c5c2526568c19f31d2c55d4c0b427/src/webrtc/groupCall.ts#L291-L305